transmission-daemon@patapon.info/extension.js

```diff
@@ -467,33 +464,33 @@ const TransmissionDaemonIndicator = new Lang.Class({
         this.setMenu(menu);
 
         this.updateOptions();
-        let settingsId = gsettings.connect('changed', Lang.bind(this, function() {
+        let settingsId = gsettings.connect('changed', () => {
             this.updateOptions();
             this.updateStats(true);
-        }));
+        });
 
-        this.connect('destroy', Lang.bind(null, function() {
+        this.connect('destroy', () => {
             gsettings.disconnect(settingsId);
-        }));
+        });
 
         this.refreshControls(false);
 
         if (gsettings.get_enum(TDAEMON_TORRENTS_DISPLAY) == TorrentDisplayClass.TransmissionTorrentSmall) {
             this.toggleDisplayMode(true);
         }
-    },
+    }
 
-    hide: function() {
+    hide() {
         if (!this._always_show) {
             this.actor.hide();
         }
-    },
+    }
 
-    show: function() {
+    show() {
         this.actor.show();
```

Comment:
I get this error, unless I rem this line out:
Extension "transmission-daemon@patapon.info" had error: InternalError: too much recursion